setup.cfg

```diff
@@ -93,6 +93,10 @@ skip=.tox,
      pootle/apps/pootle_store/views.py,
      pootle/core/search/backends/elasticsearch.py,
      pootle/core/utils/version.py,
+     pootle/core/views.py,
+     pootle/apps/pootle_language/urls.py,
+     pootle/apps/pootle_project/urls.py,
+     pootle/apps/pootle_translationproject/urls.py,
```

Comment:
its throwing an error because of indentation, so i fixed the ordering and at @dwaynebailey 's suggestion added the files to the skiplist

I also cloned the isort repo to see how easy it would be to make this test work for us, but havent had time to follow up